WcaOnRails/db/migrate/20240206154319_migrate_seniors_to_roles.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class MigrateSeniorsToRoles < ActiveRecord::Migration[7.1]
+  def change
+    ActiveRecord::Base.transaction do
+      User.where(delegate_status: 'senior_delegate').each do |senior_delegate|
+        region = UserGroup.find(senior_delegate.region_id)
+        raise "Region not found for senior_delegate: #{senior_delegate.name}" unless region
+
+        metadata = RolesMetadataDelegateRegions.create!(status: RolesMetadataDelegateRegions.statuses[:senior_delegate])
+        UserRole.create!(
+          user_id: senior_delegate.id,
+          group_id: senior_delegate.region_id,
+          start_date: Date.today - 10.years, # For now, making the start date as 10 years before for everyone. Jacob will update the correct start date for each senior delegate after migration.
```

Comment:
Can you make this a fixed date. like the founding date of the WCA?
This will make the migrated entries much easier to spot. Otherwise, you need to know when the migration was run to know which fake start_dates to look for.